jre_emul/Classes/com/google/j2objc/security/cert/IosX509Certificate.java

```diff
@@ -142,107 +184,139 @@ public void verify(PublicKey key, String sigProvider)
     throw new AssertionError("not implemented");
   }
 
-  // The X509 certificate properties are not available from the iOS Security
-  // Framework API. To get this properties would require an ASN.1 decoder, so
-  // only implement these when they are required.
+  // The X509 certificate properties are indirectly available from the iOS Security
+  // Framework API. To get this properties would require an ASN.1 decoder
+  // We have these things since commit fa9c48a1 and 592382e0
+  // by using harmony's TBSCertificate, Certificate, etc.
 
```

Comment:
First sentence is fine. Simplify rest of comment to something like: "The ASN.1 decoder from Apache Harmony is used to expand the raw format returned by the Security Framework."